packages/core/src/lib/toast/variants.ts

```diff
@@ -1,10 +1,29 @@
-import type { IconName } from '$lib/icon/icons';
-
 export const ToastVariant = {
```

Comment:
Avoid combining external details (a toast variant) with internal details (e.g. what tailwind class we apply for a given variant). This should be two objects, on public and one internal:

```ts
export const ToastVariant = {
  SUCCESS: 'success',
  // ...
} as const;

export const DisplayDetailsByVariant = {
  [ToastVariant.SUCCESS]: {
    icon: 'information-outline',
    iconClasses: 'text-info-dark',
    label: 'info',
  },
  // ...
} as const;
```